src/applications/vaos/covid-19-vaccine/components/PlanAheadPage.jsx

```diff
@@ -36,6 +36,10 @@ export default function PlanAheadPage() {
           If you’re eligible for a booster shot or additional dose, contact your
           VA health facility.
         </li>
+        <li>
```

Comment:
This looks to be a duplicate bullet item from above

Reply:
Thanks. I don't know how that got there!